### PR TITLE
[MDS-52] Updated function after testing it against prod data

### PR DESCRIPTION
**Pull request by lipsotiko**

### CODE REVIEW

File: TariffRatesBlobTrigger/translator.py [hunk 1]
```diff
@@ -49,15 +49,17 @@ def format_publishing_metadata(publishing_docs):
 
 
 def dict_list_to_csv(data, max_urls):
-    tmp_row = data[0]
-    for i in range(max_urls):
+    row_keys = []
```
Comment (loren) [inline TariffRatesBlobTrigger/translator.py:52]: would this also work? `row_keys = list(data[0].keys())`

File: TariffRatesBlobTrigger/translator.py [hunk 2]
```diff
@@ -49,15 +49,17 @@ def format_publishing_metadata(publishing_docs):
...
+    for k in data[0].keys():
+        row_keys.append(k)
+    for i in range(max_urls + 1):
         url_header = get_url_header(i)
-        if url_header not in tmp_row.keys():
-            tmp_row[url_header] = None
+        if (url_header not in row_keys):
```
Comment (loren) [inline TariffRatesBlobTrigger/translator.py:57]: Need the parens?

File: TariffRatesBlobTrigger/translator.py (hunk 1 above)
Comment (lipsotiko) [reply TariffRatesBlobTrigger/translator.py:52]: Yup! Thanks, I updated it.

File: TariffRatesBlobTrigger/translator.py (hunk 2 above)
Comment (lipsotiko) [reply TariffRatesBlobTrigger/translator.py:57]: Nope, that's what happens when switching from Java to Python :) I updated it.

**Pull request merged**